cmd/cli/multiregion/setup.go

```diff
@@ -184,14 +184,12 @@ func setMultiregionVariables(pFlags PersistentFlags) {
 	setVars(pFlags, pmaSecondaryWorkspace(pFlags), pmaVars)
 
 	emailVars := []lib.TFVar{
-		{Key: "email_subdomain", Value: pFlags.idp + "-email-secondary"},
```

Comment:
I see you left pma_subdomain in there with `-secondary`.  Just checking if that's right.

Reply:
Yes, I want both regions to be able to start up pma. And since the URL isn't referenced anywhere else in the system, it can be anything.
Good question.